_commands/basics/who.md

```diff
@@ -3,22 +3,65 @@
 
 who
 --
-TODO: Add documentation for this command by submitting a pull request.
 <!-- one line explanation would go here -->
+## Description
+Displays list of all users currently logged on 
 
 <!-- minimal example -->
+### Example Output 
 ~~~ bash
-*short* example of command usage and output
+$ Username console  Oct 12 08:20 
```

Comment:
The `$ Username ...` output doesn't need `$` in front of it. But there does need to be a `$ who` added before this output so people know where this output is coming from.